### PR TITLE
Edit/zooming images contd

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [X] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

#545 

### Please describe the goal of this pull request and the changes that were made.

This continues the work that was done yesterday. I have both edited some of the content but also added new content related to adding external IIIF images. 

### Additional Comments

@geealbers a few questions/considerations:

1. Not sure if the screenshots that I included are helpful or make sense.
2. I'm unsure if including Liz Fischer's tool is okay. Dot at UPenn is the one who told me about the tool on my call with her today. It eliminates the need for SO much explanation around finding the `canvasId`, which is handled drastically differently by each institution I looked at (Harvard, AIC, and Getty.) Happy to discuss further.
3. Does this seem sufficient for now? I know we have a lot of improvements on the way, but I think those should be added as they become available. Anything else critical I appear to be missing?

As a side note, I did a test including the external images. I came across one weird glitch that I can show you on Monday. Thanks!

### CODE REVIEW

File: content/docs-v1/zooming-images.md [hunk 1]
```diff
@@ -57,21 +57,41 @@ To speed development time, Quire will skip processing zooming images that have p
...
+
+{{< q-figure id="iiif-manifest" >}}
+
+The next attribute you need to include in the `figures.yaml` is the `canvasId`. Every institution's manifest is formatted differently so locating the `canvasID` can be a bit tricky. The easiest way to find it is with a tool like Liz Fischer's [IIIF Manifest Explorer](https://www.lizmfischer.com/iiif-tools/manifest). If you plug the manifest URL (`manifestId`) into the URL bar, the tool will breakdown all the metadata in the manifest so it is easier to read, including identifying the `canvasID` which you will then add to your `figures.yaml` entry.
+
+{{< q-figure id="liz-fischer" >}}
 
-## Use Existing External IIIF Images
+In the `figures.yaml` you will also need to include a `printImage` attribute and a `src` attribute. These point to where you have saved your image in your project. So for the example above, `irises.jpg` is saved in the projects `_assets/images/figures` folder. The `src` value will find the image and add it to your Markdown page and the `printImage` will make sure that image is included in the PDF/print version of your publication.
```
Comment (geealbers) [inline content/docs-v1/zooming-images.md:93]: I'm not sure what you mean by "The `src` value will find the image and add it to your Markdown page."?

Also, should we acknowledge that it's sort of silly to have to point to the static images twice, but that we'd like to fix it someday?

Also, for people who know IIIF, should we point out that they can't use an external URL for either `src` or `printImage` (like https://media.getty.edu/iiif/image/e5d29650-11f8-4897-9540-54a9dd65b04f/full/900,/0/default.jpg)? This is the issue that Rachel at Nelson-Atkins ran into, I think.

File: content/docs-v1/zooming-images.md [hunk 2]
```diff
@@ -57,21 +57,41 @@ To speed development time, Quire will skip processing zooming images that have p
...
+```
+
+The `manifestId` or manifest URL is a link to a JSON file that contains important metadata about an object including things like artist, name, and copyright information. A manifest URL will always accompany a IIIF image, however, each institution displays this information a bit differently. Sometimes there is a IIIF button that will take you to a manifest, other institutions might include the manifest URL below or alongside the tombstone information on an object's collection page. Here is an example of where the manifest URL is found on the Getty collection page for Van Gogh's *Irises*:
+
+{{< q-figure id="iiif-manifest" >}}
+
+The next attribute you need to include in the `figures.yaml` is the `canvasId`. Every institution's manifest is formatted differently so locating the `canvasID` can be a bit tricky. The easiest way to find it is with a tool like Liz Fischer's [IIIF Manifest Explorer](https://www.lizmfischer.com/iiif-tools/manifest). If you plug the manifest URL (`manifestId`) into the URL bar, the tool will breakdown all the metadata in the manifest so it is easier to read, including identifying the `canvasID` which you will then add to your `figures.yaml` entry.
```
Comment (geealbers) [inline content/docs-v1/zooming-images.md:89]: IIIF Manifest Explorer??!! Great find!

File: content/docs-v1/zooming-images.md [hunk 3]
```diff
@@ -57,21 +57,41 @@ To speed development time, Quire will skip processing zooming images that have p
...
+
+{{< q-figure id="liz-fischer" >}}
 
-## Use Existing External IIIF Images
+In the `figures.yaml` you will also need to include a `printImage` attribute and a `src` attribute. These point to where you have saved your image in your project. So for the example above, `irises.jpg` is saved in the projects `_assets/images/figures` folder. The `src` value will find the image and add it to your Markdown page and the `printImage` will make sure that image is included in the PDF/print version of your publication. 
 
-Coming Soon!
\ No newline at end of file
+{{< q-class "box tip" >}}
+- At the moment the +/- signs typically used to zoom in and out of an image are not available in Quire. Instead zooming can be achieved by scrolling or pinching. Adding these controls back to Quire is part of the work laid out in [GitHub issue #815](https://github.com/thegetty/quire/issues/815) mentioned above.
```
Comment (geealbers) [inline content/docs-v1/zooming-images.md:96]: Positioned here, this feels like it's related to external IIIF images specifically. Should it maybe move up to the first section of the page?

**Comment by Erin-Cecele** [comment]
Thanks @geealbers. Updates have been made.

**Review by geealbers — APPROVED** [review]

**Pull request merged**